toolchain/parser/parse_node_kind.def

```diff
@@ -282,14 +282,18 @@ CARBON_PARSE_NODE_KIND_BRACKET(StructTypeLiteral,
                                StructLiteralOrStructTypeLiteralStart)
 
 // `interface`:
-//   _external_: DeclaredName
-//     InterfaceBodyStart
-//     _external_: statements
-//   InterfaceBodyEnd
+//     InterfaceIntroducer
+//     DeclaredName
+//   InterfaceDefinitionStart
+//   _external_: declarations
 // InterfaceDefinition
-CARBON_PARSE_NODE_KIND_CHILD_COUNT(InterfaceBodyStart, 0)
-CARBON_PARSE_NODE_KIND_BRACKET(InterfaceBody, InterfaceBodyStart)
-CARBON_PARSE_NODE_KIND_BRACKET(InterfaceDefinition, DeclaredName)
+//
+// The above is a definition; for a declaration, InterfaceDeclaration will end
+// it InterfaceDefinitionStart is for a definition.
```

Comment:
I can't parse this comment.

Reply:
Trying to rephrase to help, along with the matching function comment. New text:

```
// The above is the structure for a definition; for a declaration,
// FunctionDefinitionStart and later nodes are removed and replaced by
// FunctionDeclaration.
```